lib/smart_todo/events/gem_bump.rb

```diff
@@ -17,16 +17,17 @@ class GemBump
       #
       # @example Expecting a version in the 5.x.x series
       #   GemBump.new('rails', ['> 5.2', '< 6'])
-      def initialize(gem_name, requirements)
+      def initialize(gem_name, requirements, spec_set = Bundler.load.specs)
```

Comment:
Do we use this class without the last argument?

Reply:
Nope, removed the class wholesale.